ui-core/src/participant/landing/sections/HeroWithImageTemplate.tsx

```diff
@@ -121,14 +168,14 @@ function HeroWithImageTemplate(props: HeroWithImageTemplateProps) {
               'd-flex justify-content-center align-items-center p-0'
             )}
           >
-            <ConfiguredMedia media={image} className="img-fluid"/>
+            {!imageAsBackground && <ConfiguredMedia media={image} className="img-fluid"/>}
           </div>
         )}
         <div
           className={classNames(
             'col-12', `col-lg-${12 - imageCols}`,
             'py-3 p-sm-3 p-lg-5',
-            'd-flex flex-column flex-grow-1 justify-content-around'
+            'd-flex flex-column flex-grow-1 justify-content-center'
```

Comment:
This should be the only change that actually effects how things look for production user. I think `justify-content-center` looks nicer in general (and especially for the AT landing page), and it should be a minor change of just a few pixels of spacing for the most part, but we should definitely go through all of the portals in detail to make sure the changes don't look off.